universal-application-tool-0.0.1/app/views/dev/AzureViewStorageStrategy.java

```diff
@@ -0,0 +1,48 @@
+package views.dev;
+
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.input;
+import static j2html.TagCreator.text;
+
+import j2html.TagCreator;
+import j2html.tags.ContainerTag;
+import services.cloud.StorageUploadRequest;
+import services.cloud.azure.BlobStorageUploadRequest;
+import views.HtmlBundle;
+import views.ViewUtils;
+
+/** Strategy class for creating a file upload form for Azure. */
+public class AzureViewStorageStrategy implements FileUploadViewStorageStrategy {
+
+  private static final String AZURE_STORAGE_BLOB_WEB_JAR =
+      "lib/azure__storage-blob/browser/azure-storage-blob.min.js";
+
+  @Override
+  public ContainerTag getFileUploadForm(
+      ViewUtils viewUtils, StorageUploadRequest storageUploadRequest, HtmlBundle bundle) {
+    if (!(storageUploadRequest instanceof BlobStorageUploadRequest)) {
+      return null;
+    }
```

Comment:
suggestion (blocking): avoid returning null or assigning null to variables. If a value may not be present change the return type to Optional and return an Optional.empty()

question (blocking): Is hitting this conditional an expected part of user flow or is it guarding an error case? if the latter please log an error before returning or throw an exception. if the former please add a comment.

Reply:
It is not expected, so I've updated the code to throw an exception